recipes/fc-virus/meta.yaml

```
@@ -0,0 +1,38 @@
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - make
+  host:
+    - make
+    - boost
+    - zlib
+    - htslib
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)